src/input_handlers.h

```diff
@@ -201,20 +161,20 @@ class VmaZCopyReadInputHandler : public MessageParser<BufferAccumulation> {
             return 0;
         }
         // Receive the next packet with zero copy API
-        ret = g_vma_api->recvfrom_zcopy(fd, m_ptr->m_pkt_buf, Message::getMaxSize(), &flags,
-                                        (struct sockaddr *)recvfrom_addr, &size);
+        ret = recvfrom_zcopy(fd, m_ptr->m_pkt_buf, Message::getMaxSize(), &flags,
+                             (struct sockaddr *)recvfrom_addr, &size);
 
         if (likely(ret > 0)) {
             if (flags & MSG_VMA_ZCOPY) {
```

Comment:
It is based on assumption that `MSG_VMA_ZCOPY == MSG_XLIO_ZCOPY`. It is a risky. Consider more stable solution